concepticondata/concepticon.tsv

```diff
@@ -1942,7 +1942,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1941	SOLDIER	Warfare and hunting	A person who is actively engaged in battle, conflict or warfare.	Person/Thing	
 1942	NEAR	Spatial relations	Not far distant in time or space or degree or circumstances.	Property	
 1943	CROWN (OF HEAD)	The body	A royal, imperial or princely headdress; a diadem.	Person/Thing	
-1944	GOD	Religion and belief	A supernatural, typically immortal being with superior powers.	Person/Thing	
+1944	GOD	Religion and belief	The deity of a monotheistic religion.	Person/Thing	
```

Comment:
Considering the words we have (even from IE in NorthEuralex and everything), the original is the correct. While most data might be collected referring to Abrahamic religions, this is just an obvious colexification. I'd strongly suggest to keep the original here.

Reply:
See comment below.